.changelog/12573.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+rpc: (beta-only): add a new metric `consul.rpc.server.call` with labels
```

Comment:
Why "beta only"?

Reply:
hm, I was marking that this is coming out in `Beta` mode? 🤔 Would there be another way to mark it as such?